samples/node/index.js

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env node
+
+const { initWasm, TW } = require("@trustwallet/wallet-core");
+
+(async function () {
```

Comment:
Do we need to have this example in JS taking into account that we have `index.ts`?